### ISSUE TITLE
Investigate running HTTPD in Kubernetes with SCL3-derived Apache config

### ISSUE THREAD

**Issue by metadave** [mozmar/infra#181]
[Parent issue](https://github.com/mozmar/infra/issues/147)

Once we have the [SCL3 Apache config](https://bugzilla.mozilla.org/show_bug.cgi?id=1357511), determine:

- can we run HTTPD in Kubernetes with an SCL3-derived Apache config?
- the amount of effort required
- technical debt carried forward

**Comment by metadave** [comment]
Here's a starting point w/ an older version of httpd:

```
FROM httpd:2.2
COPY ./public-html/ /usr/local/apache2/htdocs/
```

It works fine with a simple deployment:

```
apiVersion: extensions/v1beta1
kind: Deployment
metadata:
  name: mdn-httpd
spec:
  replicas: 1
  template:
    metadata:
      labels:
        app: mdn-httpd
    spec:
      containers:
      - name: mdn-httpd
        image: quay.io/metadave/mdn-httpd:latest
        ports:
        - containerPort: 80
---
kind: Service
apiVersion: v1
metadata:
  name: mdn-httpd
spec:
  selector:
    app: mdn-httpd
  type: LoadBalancer
  ports:
    - protocol: TCP
      port: 80
```

**Comment by jgmize** [comment]
See https://bugzilla.mozilla.org/show_bug.cgi?id=1357511#c1 for links to SCL3 apache config and docs, and that older version is exactly what we need, as the nodes in scl3 are running 2.2.

**Comment by metadave** [comment]
tl;dr: it's not worth it. See [here](https://github.com/mozmar/infra/issues/180)
